hammer/par/openroad/__init__.py

```diff
@@ -188,12 +204,32 @@ def open_chip_tcl(self) -> str:
         return self.open_chip_script + ".tcl"
 
     @property
-    def openroad_log(self) -> str:
+    def openroad_latest_log(self) -> str:
         return os.path.join(self.run_dir,"openroad.log")
 
     @property
     def create_archive_mode(self) -> str:
-        return self.get_setting("par.openroad.create_archive_mode")
+        return self.get_setting('par.openroad.create_archive_mode')
+
+    @property
+    def reports_dir(self) -> str:
+        return os.path.join(self.run_dir, "reports")
+
+    @property
+    def clock_port_name(self) -> str:
+        clock_port = self.get_clock_ports()[0]
+        return clock_port.name
+
+    @property
+    def metrics_dir(self) -> str:
+        return os.path.join(self.run_dir, "metrics")
+
+    @property
+    def metrics_file(self) -> str:
+        return os.path.join(self.metrics_dir, f"metrics-{self._steps_to_run[-1]}.json")
```

Comment:
What happens if using step control we don't get to these last steps that can have metrics (since `-metrics` seems to be a cmdline flag)?

Reply:
Just checked and it outputs an empty JSON. Which should be fine as a placeholder, since there technically could be metrics after any step of the flow.